core/store/src/lib.rs

```diff
@@ -291,28 +291,79 @@ impl NodeStorage {
     }
 }
 
+pub fn retrieve_shard_uid_from_db_key(key: &[u8]) -> io::Result<ShardUId> {
```

Comment:
Just to make sure I understand - this method is here just to map the error right? 

nit: parse_shard_uid

Reply:
the method was removed in next commit